scompose/project/project.py

```diff
@@ -254,7 +276,7 @@ def create_hosts(self, lookup):
 
```

Comment:
Expected: When I run 
```
singularity-compose up <container1>
singularity-compose up <container2>
```
If container 1 and 2 are in the same compose file and on the same network (both sudo or both fakeroot), they should be able to communicate via hostname.

Actual Behavior:
The etc.hosts file only adds IP-hostname mappings for the most recent entry, so inter-container comms via hostname fails.

Suggested Remedy:
Search for running containers from the same yaml file on the network for the container just spun up and add them to this list of lookup items.

Reply:
This should be fixed.

Reply:
Yes, I've verified this is working on my side too